src/apps/external_executor/external_executor.cpp

```diff
@@ -80,7 +79,10 @@ namespace externalexecutor
         ExecutorNodeInfo executor_info = {
           executor_public_key, payload.attestation(), supported_endpoints};
 
-        ExecutorIDs[executor_id] = executor_info;
+        executor_ids[executor_id] = executor_info;
+
+        // Record the cert in the Executor certs map
+        executor_certs[executor_id] = executor_public_key;
```

Comment:
Map is called `_certs` but we record a public key in it. Could the map be renamed?

Reply:
Since it now stores the `cert` instead, renamed it back